pkg/pillar/cmd/monitor/messages.go

```diff
@@ -92,3 +100,36 @@ func (ctx *monitor) sendAppsList() {
 	}
 	ctx.IPCServer.sendIpcMessage("AppsList", apps)
 }
+
+func readEfiVars(fsys fs.FS) ([]efiVariable, error) {
+	vars, err := fs.ReadDir(fsys, ".")
+	if err != nil {
+		return nil, err
+	}
+
+	// read boot order
+	bootOrder, err := fs.ReadFile(fsys, "BootOrder")
+	if err != nil {
+		return nil, err
+	}
+
+	re := regexp.MustCompile(`^Boot[0-9a-fA-F]{4}$`)
```

Comment:
@eriknordmark would definitely say that this could be compiled at package initialization and not every time the function is called :)

Reply:
Fixed